packages/grafana-ui/src/components/GraphNG/GraphNG.tsx

```diff
@@ -159,15 +151,13 @@ export const GraphNG: React.FC<GraphNGProps> = ({
       {(vizWidth: number, vizHeight: number) => (
         <UPlotChart
           data={alignedData}
+          config={currentConfig}
```

Comment:
Not sure what is best, but I was thinking that the builder (our config object) object was the public interface for UPlotChart and that UPlotChart still did not expose any UPlot types. At least if we plan to expose UPlotChart directly to plugins

Reply:
this is done now